Gems/Atom/RHI/DX12/Code/Source/RHI/FrameGraphCompiler.cpp

```diff
@@ -468,8 +466,6 @@ namespace AZ
             ResourceTransitionLoggerNull logger(imageFrameAttachment.GetId());
 #endif
 
-            AZ_PROFILE_SCOPE(RHI, "FrameGraphCompiler: CompileImageBarriers (DX12)");
-
```

Comment:
I disagree with this one, I see events lasting up to 35us

Reply:
Will leave this one and the buffer barriers one in then